src/Meditation.UnitTests/AttachToProcess/AttachableProcessListProviderTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Meditation.Common.Services;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Meditation.UnitTests.AttachToProcess
+{
+    public class AttachableProcessListProviderTests : TestsBase
```

Comment:
[Suggestion] I guess it's better to make a test project for the every assembly in solution.

Reply:
I added this to #11. It would make sense to do it together